src/SamplesApp/SamplesApp.Shared/App.xaml.cs

```diff
@@ -129,11 +129,53 @@ override void OnLaunched(LaunchActivatedEventArgs e)
 			DisplayLaunchArguments(e);
 		}
 
+		protected
+#if HAS_UNO
+			internal
+#endif
```

Comment:
I'm curious... why do we need this?

Reply:
Otherwise it throws a compilation error, I guess due to the fact Uno.UI has InternalsVisibleTo for SamplesApp, so it has to be "qualified" in full.